tests/test_open.py

```diff
@@ -226,6 +226,9 @@ def test_node_round_trip(tmp_path, node_class):
 @pytest.mark.filterwarnings("ignore:This function assumes shape is 2D")
 @pytest.mark.filterwarnings("ignore:Input shape must be 5D")
 def test_opening_model(tmp_path, node_class):
+    if node_class == stnode.SourceCatalog:
+        pytest.xfail("SourceCatalog does not have a meta attribute yet")
+
```

Comment:
Looks like your wanting to include a meta attribute in `SourceCatalog` if this is the case, update your schemas to include that and remove this change. I think that will fix many of the issues.

Reply:
It seems like there's some uncertainty on whether or not to include it, and definitely uncertainty on what that meta should contain. I got the tests passing for now, but I agree that this bit may be unnecessary now.